NetKVM/Common/virtio_net.h

```diff
@@ -1,134 +1,137 @@
 #ifndef _LINUX_VIRTIO_NET_H
```

Comment:
@YanVugenfirer @ybendito Is this file from an external repo like a Linux kernel? 

@Jedoku If yes, exclude this file from clang-format.